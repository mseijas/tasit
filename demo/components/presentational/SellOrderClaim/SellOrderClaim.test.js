import React from "react";
import { shallow } from "enzyme";
import SellOrderClaim from "./SellOrderClaim";

describe("SellOrderClaim", () => {
  it("renders the component", async () => {
    expect(
      shallow(<SellOrderClaim sellOrder={{}} onClaim={() => {}} />)
    ).toMatchSnapshot();
  });
});
