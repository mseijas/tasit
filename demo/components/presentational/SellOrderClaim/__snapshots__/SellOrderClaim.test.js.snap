// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrderClaim renders the component 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <SellOrder
    sellOrder={Object {}}
  />
  <Component
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 66.7,
      }
    }
  >
    <Button
      onPress={[Function]}
      title="Claim"
    />
  </Component>
</Component>
`;
